PyTorch/Segmentation/MaskRCNN/pytorch/tools/train_net.py

```diff
@@ -109,21 +115,14 @@ def train(cfg, local_rank, distributed, fp16, dllogger):
         model, optimizer = amp.initialize(model, optimizer, opt_level=amp_opt_level)
 
     if distributed:
-        if use_apex_ddp:
-            model = DDP(model, delay_allreduce=True)
-        else:
-            model = torch.nn.parallel.DistributedDataParallel(
-                model, device_ids=[local_rank], output_device=local_rank,
-                # this should be removed if we update BatchNorm stats
-                broadcast_buffers=False,
-            )
+        model = DDP(model, device_ids=[local_rank], broadcast_buffers=False, bucket_cap_mb=bucket_cap_mb)
```

Comment:
Please, leave some comment around `broadcast_buffers`, so we can trace that this behavior is different from default repo